env.yaml

```diff
@@ -13,6 +13,8 @@ dependencies:
   - pytorch
   - seaborn
   - pyarrow
+  - numba
```

Comment:
Consider specifying which channels these packages should come from as well.

Reply:
resolved